.gitattributes

```diff
@@ -0,0 +1,2 @@
+# Prevent git from normalizing or converting any newline or encoding inside test data folders

```

Comment:
Need to update the comment that we want to keep yaml and json files with the original line endings (LF) to simplify testing.